src/dawn/engine_dawn.h

```diff
@@ -73,12 +73,18 @@ class EngineDawn : public Engine {
   // on the device, the buffer on the host that will eventually hold the
   // resulting pixels for use in checking expectations, and bookkeeping info
   // for that host-side buffer.
-  Result CreateFramebufferIfNeeded(RenderPipelineInfo* render_pipeline);
-
-  // TODO(dneto): Remove this. Shaders are attached to the pipeline.
-  Result SetShader(ShaderType type, const std::vector<uint32_t>& data);
+  Result AttachBuffersAndTextures(RenderPipelineInfo* render_pipeline);
 
   ::dawn::Device* device_ = nullptr;  // Borrowed from the engine config.
+  ::dawn::Buffer fb_buffer_;
+  bool fb_is_created_ = false;
+
+  ::dawn::TextureView texture_view_;
```

Comment:
I assume texture_view_ is the view into fb_texture_ ?  If so then I'd put the fb_texture_ member first, then texture_view_ second and add a comment saying that it's a view into fb_texture_